e2e/cypress/integration/system_console/unsaved_changes_spec.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+
+describe('Unsaved Changes', () => {
+    before(() => {
+        cy.apiAdminLogin();
+    });
```

Comment:
nit: This can be safely removed since up until here, user session is still with the sysadmin.